app/assets/stylesheets/leaderboard.css

```diff
@@ -0,0 +1,282 @@
+/* variables */
+:root {
+  --gradient-top-three: var(--gradient-blue-green);
+  --gradient-others: linear-gradient(to right, red, gold);
+  --gradient-row-glare: linear-gradient(to right, #ffffff00, #ffffff40, #ffffff00);
```

Comment:
I think the gradient bars look a little dark, like they 'glow' less than the figma designs. I'm not sure what the best way to lighten them is, either changing the colours or changing the glow?

Reply:
This is an inconsistency within Figma. I changed part of the bars to this dark version in the Figma file and missed some others. This dark version is using the Alces Flight theme color as the blue part and has a higher contrast on the colored background.

However, following your comment, I've pushed a new commit using the brighter one. It's up to you to evaluate between these two versions before it is getting approved.